akka-remote/src/main/scala/akka/remote/artery/tcp/TcpFraming.scala

```diff
@@ -0,0 +1,101 @@
+/**
+ * Copyright (C) 2018 Lightbend Inc. <http://www.lightbend.com>
+ */
+package akka.remote.artery
+package tcp
+
+import java.nio.ByteBuffer
+import java.nio.ByteOrder
+
+import akka.annotation.InternalApi
+import akka.remote.artery.FlightRecorderEvents.TcpInbound_Received
+import akka.stream.Attributes
+import akka.stream.impl.io.ByteStringParser
+import akka.stream.impl.io.ByteStringParser.ByteReader
+import akka.stream.impl.io.ByteStringParser.ParseResult
+import akka.stream.impl.io.ByteStringParser.ParseStep
+import akka.stream.scaladsl.Framing.FramingException
+import akka.stream.stage.GraphStageLogic
+import akka.util.ByteString
+
+/**
+ * INTERNAL API
+ */
+@InternalApi private[akka] object TcpFraming {
+  val Undefined = Int.MinValue
+
+  /**
+   * The first 4 bytes of a new connection must be these `64x0 75x0 75x0 64x0` (AKKA).
```

Comment:
Nice ;)

I guess a magic code like this needs to be approved by @ktoso. Not that people that see it in the wild and use this spelling in presentations...

Btw. isn't it usually `0x64` instead of `64x0`.

Reply:
thx, typo in comment

Reply:
> I guess a magic code like this needs to be approved by @ktoso. 

LOL ;-) I'm okey with it ;-)